### ISSUE TITLE
Module not found error occured when npm install on Windows 10

### ISSUE THREAD

**Issue by jackey** [conversejs/converse.js#2043]
**Describe the bug**

I was just easily try to clone and npm install on windows pc .

```
git clone https://github.com/conversejs/converse.js.git conversejs
cd conversejs &&  npm install 
```
the error is like and many
```
ERROR in ./src/templates/form_input.html
Module not found: Error: Can't resolve 'Dappconversejs
ode_moduleslodashescape.js' in 'D:\app\conversejs\src\templates'
 @ ./src/templates/form_input.html 1:16-107
 @ ./src/converse-register.js
 @ ./src/converse.js
 @ ./src/entry.js
```
 i followed the document of install 3rd development libraries :) 
maybe webpack configuration wrong ?

**Comment by jcbrand** [comment]
We use GNU Make to build files (see the Makefile) and making builds on Windows is not supported.

I think the issue here is that Lerna needs to be run first.

You can try the following:

	npm run lerna
	npm rebuild node-sass
        npm run dev

**Comment by ch34** [comment]
> We use GNU Make to build files (see the Makefile) and making builds on Windows is not supported.
> 
> I think the issue here is that Lerna needs to be run first.
> 
> You can try the following:
> 
> ```
> npm run lerna
> npm rebuild node-sass
> npm run dev
> ```

Hi jcbrand, I tried your command above to build conversejs on windows but still having the same issue. I am currently building off the latest master branch of conversejs repository. Has anyone been able to build on Windows successfully or at this time it can only be built on linux. Thanks.

**Comment by jcbrand** [comment]
@ch34: There was a bug in the releases. It should be fixed now with version 7.0.2.

As a workaround you can also set your package.json to check out a specific commit of Converse instead of a release tag.

**Comment by MayurKJParekh** [comment]
With 7.0.2 on Windows 10 
Getting error to build 

ERROR in ./images/icons.svg
Module not found: Error: Can't resolve 'C:UsersAdminDownloadsconverse.js-masterconverse.js-master
ode_moduleslodashescape.js' in 'C:\Users\Admin\Downloads\converse.js-master\converse.js-master\images'
 @ ./images/icons.svg 1:16-119
 @ ./src/components/font-awesome.js
 @ ./src/templates/converse.js
 @ ./src/converse-chatboxviews.js
 @ ./src/converse-chatview.js
 @ ./src/converse.js
 @ ./src/entry.js
